.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: Test
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.3
```

Comment:
I would like to upgrade Ruby version to 3 but after investigation, I noticed it needs too many changes to be reviewed at once.